client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -86,26 +89,25 @@ class EditorConfirmationSidebar extends React.Component {
 	getBusyButtonLabel( publishButtonStatus ) {
 		switch ( publishButtonStatus ) {
 			case 'update':
-				return this.props.translate( 'Updating...' );
+				return this.props.translate( 'Updating…' );
 			case 'schedule':
-				return this.props.translate( 'Scheduling...' );
+				return this.props.translate( 'Scheduling…' );
 			case 'publish':
-				return this.props.translate( 'Publishing...' );
+				return this.props.translate( 'Publishing…' );
 			case 'requestReview':
-				return this.props.translate( 'Submitting for Review...' );
+				return this.props.translate( 'Submitting for Review…' );
 		}
 
-		return this.props.translate( 'Publishing...' );
+		return this.props.translate( 'Publishing…' );
 	}
 
 	renderPrivacyControl() {
 		if ( ! this.props.post ) {
 			return;
 		}
 
-		const { password, type } = this.props.post || {};
+		const { password, type, isPrivateSite } = this.props.post || {};
```

Comment:
Beware: `isPrivateSite` is a property of `this.props`, not `this.props.post`.

Reply:
Great catch, thanks!